bcgov/biohubbc-platform
# BHBC 1903

# Overview

## Links to Jira tickets

[BHBC 1903](https://quartech.atlassian.net/browse/BHBC-1903?atlOrigin=eyJpIjoiMzBjNDQ0Y2FmMWQwNDhmYmE0YmJlZTFjMGRhOTI0YTQiLCJwIjoiaiJ9)

## Description of relevant changes

Updated data set search endpoint to return number of occurrences based on users security.
To Test: 
- navigate and try Find Dataset search as a public user (not logged in), track the number of occurrences
- login and try the Find Dataset search and track number of occurrences (should be more)

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
